includes/Core/Authentication/Authentication.php

```diff
@@ -1455,7 +1455,7 @@ private function cron_update_remote_features() {
 	 *
 	 * @param string $action Action name.
 	 */
-	public static function invalid_nonce_error( $action ) {
+	public function invalid_nonce_error( $action ) {
```

Comment:
Rather than changing this function to instance method, can't we just pass the `support_link_url` as a parameter? That seems cleaner and less noisy. What do you think?

Reply:
@kuasha420, in that case, we will have to make the `get_proxy_support_link_url` method a public method. Or we will have to create a new public method that returns the concatenated url like so:
```php
public function get_invalid_nonce_support_link_url() {
	return esc_url( $this->get_proxy_support_link_url() . '?error_id=nonce_expired' );
}
```

Which would be better, changing `invalid_nonce_error` to a non-static or the above?

Reply:
@hussain-t Great point. I just realized that the `invalid_nonce_error` is also being used outside the class and your current solution is better for that approach. This LGTM. :heavy_check_mark:

Reply:
Thanks, @kuasha420